bundles/automation/org.eclipse.smarthome.automation.parser.json/src/main/java/org/eclipse/smarthome/automation/parser/internal/json/ModuleJSONParser.java

```diff
@@ -372,14 +372,24 @@ private static void moduleToJSON(Module module, OutputStreamWriter writer) throw
                 }
                 int index = jsonOutput.indexOf('.');
                 if (index == -1) {
-                    JSONUtility.catchParsingException(type, UID, exceptions, new IllegalArgumentException(
-                            "Wrong format of Output : " + jsonOutput + ". Should be as \"Module_Id.Output_Id\"."), log);
-                    continue;
+                    if (jsonOutput.startsWith("$")) {
+                        String ouputModuleId = null;
+                        String outputName = jsonOutput;
+                        Connection connection = new Connection(inputName, ouputModuleId, outputName);
```

Comment:
outputModuleID is null and the validation in constructor of Connection will fail!